conf/logs.json

```diff
@@ -1666,5 +1666,40 @@
         "reason"
       ]
     }
+  },
+  "aliyun:events": {
```

Comment:
Remind to update schema name if you agree with me renaming `_type` to `actiontrail`.